### ISSUE TITLE
Output not normalized 

### ISSUE THREAD

**Issue by Oguked** [zanilzanzan/FuseNet_PyTorch#11]
I think your model is missing a normalization operation like Softmax at the end, because the output values are big positiv/negativ numbers (-42, +30 etc.).

**Comment by zanilzanzan** [comment]
Hey there,

The cross-entropy loss function I used contains softmax operation intrinsically. Please read the relevant documentation _carefully_ before opening up an issue regarding a functioning code.  

Function details: https://pytorch.org/docs/stable/nn.html#cross-entropy 
Source: torch.nn.functional.cross_entropyhttps://pytorch.org/docs/stable/_modules/torch/nn/functional.html#cross_entropy
